src/Lachain.Core/Config/ConfigManager.cs

```diff
@@ -440,6 +466,7 @@ private void _UpdateConfigToV15()
         private void _SaveCurrentConfig()
         {
             File.WriteAllText(ConfigPath, JsonConvert.SerializeObject(_config, Formatting.Indented));
+            File.WriteAllText(ConfigBackupPath, JsonConvert.SerializeObject(_config, Formatting.Indented));
```

Comment:
I think backup should be saved in e separate function after updating is done. Because the case was config was corrupted during updating the config. So, in current implementation, backup could also be corrupted, right?

Reply:
Steps during config update:
1. Copy old config to config backup
2. Write new data to config
3. Delete backup
I don't think we need separate function for this,  but step order is important

Reply:
okay, I did it slightly different way. I'm always keeping a backup file. This way, both the Config file and the backup file cannot be corrupted at the same time. In case there is a problem updating the config file, the backup file will still be intact. 

I think this solution will handle your case and also handle any case where the config file is somehow corrupted.